ada-project-docs/wave_03.md

```diff
@@ -106,6 +124,12 @@ when I send a `PATCH` request to `/tasks/1/mark_incomplete`,
 
 then I want this to behave exactly like `/tasks/1/mark_incomplete` for a complete task. Its `completed_at` value remains as `null`/`None`, and I get this response:
 
+`204 No Content`
+
+The response should have a mimetype of "application/json" to keep our API response type consistent.
+
+If after the `PATCH` request I fetch the task by sending a `GET` request to `/tasks/1`, I get this response:
```

Comment:
No need to take the suggestion, I'm just adding it again so its easier for you to commit the suggestions if you do!

```suggestion
After I have made the 'PATCH' request, I can submit a 'GET' request to 'tasks/1, which will return the response:```